extensions/objects/models/objects.py

```diff
@@ -1035,3 +1035,70 @@ class DragAndDropPositiveInt(BaseObject):
     default_value = 1
 
     SCHEMA = PositiveInt.SCHEMA
+
+
+class AlgebraicExpression(BaseObject):
+    """Class for algebraic expressions. Stores a unicode string representing a
+    valid algebraic expression.
+    """
+
+    description = 'A unicode string for an algebraic expression.'
```

Comment:
Should these be latex strings? If not, what's the full definition of the format?

Reply:
The input will be in ASCIIMath format. [More info](http://asciimath.org/).

Reply:
This info should definitely be part of the docstring.

Reply:
Added.

Reply:
Where?

Reply:
Done now, sorry!